pep8ext_naming.py

```diff
@@ -148,7 +160,7 @@ class ClassNameCheck(BaseASTCheck):
     check = MIXEDCASE_REGEX.match
     N801 = "class names should use CapWords convention"
 
-    def visit_classdef(self, node, parents):
+    def visit_classdef(self, node, parents, ignore=None):
```

Comment:
Since we're always calling the method with, do these ignore parameters need to have default values?